coco/master.py

```diff
@@ -96,19 +105,26 @@ def __del__(self):
 
         Join the worker thread.
         """
-        if self.qworker:
```

Comment:
If not checking this anymore, I guess we can remove `self.qworker = None` in the constructor.

Reply:
or actually it would be better to put this check back in because at the moment it will trip up the destructor when something goes wrong launching coco

Reply:
good catch!